### DISCUSSION TITLE
Where is path location config.yml in hyperglass?

### DISCUSSION THREAD

**Discussion by kronicklez** [thatmattlove/hyperglass#288] [Q&A] [answered]
Hi expert,

May i know the path location config.yml in hyperglass so i can change some logo and title?

Thanks and appreciate any feedback

**Comment by jdhall75** [comment, marked as answer]
@kronicklez 

The default location for the `config.yml` file is in `/etc/hyperglass`.  This can be change by setting the `HYPERGLASS_APP_PATH` environment variable.

Here are some links to help you out.
https://hyperglass.dev/installation/environment-variables
https://hyperglass.dev/configuration/config/web-ui
